Examples/RunMe-Report.ps1

```diff
@@ -1,9 +1,12 @@
-### Starts Module (Requires config above)
-Clear-Host
 Import-Module PSWinReporting -Force
 Import-Module PSSharedGoods -Force
 Import-Module PSEventViewer -Force
 
+$LoggerParameters = @{
```

Comment:
Is there a reason why this is separated? Maybe should be put back into hash that's already defined? or what's your idea for this?

Reply:
I think that the location of this setting in the "ReportOptions" section is not entirely correct. The output format of the script execution process and the path to the log are not report settings. I believe that logging settings should be higher in the hierarchy of settings and deserve a separate place, such as email settings.
I also think that the name "DisplayConsole" is less suitable for this set of options than "LoggerParameters".

Reply:
Good enough :-)